tweets/2020/semantic-release-pypi.tweet

```diff
@@ -0,0 +1,3 @@
+Behold humans! I now got the power to upload packages to PyPI.
+
+https://www.npmjs.com/package/semantic-release-pypi
```

Comment:
Everything is better with some emoji!

```suggestion
🤖📯 Behold humans! I now got the power to upload packages to PyPI.

https://www.npmjs.com/package/semantic-release-pypi
```

Do you have a twitter account? If so, we can mention it, too, if you like

Reply:
no, I don't have one